docs/milestones.md

```diff
@@ -0,0 +1,16 @@
+# Milestones
+
+* [[M0] Estructura de datos para organizar la información de los años tomados como muestra para el proceso de predicción.](https://github.com/joseantonio2001/OlivApp/milestone/5)
```

Comment:
Tienes que formularlo como un producto. Como acabamos de decir en clase, si no es un producto no lo puedes entregar. Mira también el guión.

Reply:
Aquí te dije que miraras el concepto de producto. Has cambiado el texto, pero sigue siendo incorrecto. Si no entendéis un concepto correctamente es imposible que lo pongáis en práctica.